packages/jamf_protect/data_stream/telemetry/elasticsearch/ingest_pipeline/pipeline_event_exec.yml

```
@@ -0,0 +1,198 @@
+        field: jamf_protect.telemetry.event.exec.target.is_es_client
+        target_field: custom.es_client
+        ignore_missing: true
+    - rename:
+        field: jamf_protect.telemetry.event.exec.target.cdhash
+        target_field: custom.code_directory_hash
+        ignore_missing: true
+    - rename:
+        field: jamf_protect.telemetry.event.exec.target.executable.sha1
+        target_field: process.hash.sha1
+        ignore_missing: true
+    - rename:
+        field: jamf_protect.telemetry.event.exec.target.executable.sha256
+        target_field: process.hash.sha256
+        ignore_missing: true
+    - append:
+        field: process.hash.sha1
+        value: '{{{jamf_protect.telemetry.event.exec.target.executable.sha1}}}'
+        if: ctx.jamf_protect.telemetry?.event?.exec?.target?.instigator?.executable?.sha1 != null
+        allow_duplicates: false
+        ignore_failure: true
+    - append:
+        field: related.hash
+        value: '{{{jamf_protect.telemetry.event.exec.target.executable.sha1}}}'
+        if: ctx.jamf_protect.telemetry?.event?.exec?.target?.instigator?.executable?.sha1 != null
+        allow_duplicates: false
+        ignore_failure: true
+    - rename:
+        field: jamf_protect.telemetry.event.exec.target.tty.path
+        target_field: custom.tty
+        ignore_missing: true
+    - set:
+        field: process.interactive
+        value: true
+        if: ctx.jamf_protect.telemetry?.event?.exec?.target?.tty != null
+    - set:
+        field: process.interactive
+        value: false
+        if: ctx.jamf_protect.telemetry?.event?.exec?.target?.tty == null
+    - convert:
+        field: jamf_protect.telemetry.event.exec.target.audit_token.pid
+        target_field: process.pid
+        type: long
+        ignore_missing: true
+        on_failure:
+        - append:
+            field: error.message
+            value: '{{{_ingest.on_failure_message}}}'
+    - rename:
+        field: jamf_protect.telemetry.event.exec.target.audit_token.uuid
+        target_field: process.entity_id
+        ignore_missing: true
+    - rename:
+        field: jamf_protect.telemetry.event.exec.target.executable.path
+        target_field: process.executable
+        ignore_missing: true
+    - rename:
+        field: jamf_protect.telemetry.thread.thread_id
+        target_field: process.thread.id
+        ignore_missing: true
+
+    - rename:
+        field: jamf_protect.telemetry.event.exec.args
+        target_field: process.args
+        ignore_missing: true
+        if: ctx.jamf_protect?.telemetry?.event?.exec?.args != null
+    - script:
+        description: Counts the indexes of the arguments
+        lang: painless
+        source: >
+            if (ctx.process.args instanceof List) {
+            
+            ctx.process.args_count = ((List)ctx.process.args).size();
+            } else {
+            ctx.process = ctx.process != null ? ctx.process : new HashMap();
+            ctx.process.args_count = 0;
```

```suggestion
        source: |
            if (ctx.process.args instanceof List) {
            
              ctx.process.args_count = ((List)ctx.process.args).size();
            } else {
              ctx.process = ctx.process != null ? ctx.process : new HashMap();
              ctx.process.args_count = 0;
```
I don't believe the coercion to `List` is required.